cloud/migrate-mst-cloud/schema-then-data.md

```diff
@@ -321,7 +321,14 @@ Compressed chunks in Managed Service for TimescaleDB are decompressed during the
 `COPY` operation. For more information about recompressing data, see the
 [compression section][compression].
 
+## Update table statistics
```

Comment:
Can we keep the comments out of the code blocks, please? Add any extra info to the exposition instead.

Reply:
Understood! Was this referring to a specific example? I looked for the error around here and couldn't find it, but maybe I glazed right over it

Reply:
Oh, it's possible I misread, too.

Reply:
No problem! Good to know in the future 🙂 just wanted to make sure I wasn't missing anything I had to fix.